dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentsExportController.java

```diff
@@ -133,7 +131,7 @@ PagingWrapper<ObjectNode> getEnrollments( RequestParams requestParams )
 
     @OpenApi.Response( OpenApi.EntityType.class )
     @GetMapping( value = "/{uid}" )
-    public ResponseEntity<ObjectNode> getEnrollmentByUid(
```

Comment:
why can we remove this now? Did you check that the OpenAPI spec still looks well?

The class level `@OpenApi.EntityType( Enrollment.class )` with method level `@OpenApi.Response( OpenApi.EntityType.class )` set the response type since the OpenAPI generation would not know what `ResponseEntity<ObjectNode>` is.

Reply:
I haven't check and this change wasn't suppose to be here. Reverted